catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -332,28 +344,88 @@ void registryInfoQuery() throws UnsupportedQueryException {
         SourceResponse identityMetacard = query(queryRequest);
         if (identityMetacard.getResults()
                 .size() > 0) {
-            remoteName = identityMetacard.getResults()
+            String metacardTitle = identityMetacard.getResults()
                     .get(0)
                     .getMetacard()
                     .getTitle();
+            setId(metacardTitle);
             registryId = RegistryUtility.getRegistryId(identityMetacard.getResults()
                     .get(0)
                     .getMetacard());
+            updateConfiguration(metacardTitle);
         }
-        updateConfiguration();
+
     }
 
-    private void updateConfiguration() {
+    private String getConnectionType(Bundle bundle, String pid) {
+        Locale locale = Locale.getDefault();
+        if (bundle != null) {
+            if (metaTypeService != null) {
+                MetaTypeInformation mti = metaTypeService.getMetaTypeInformation(bundle);
+                if (mti != null) {
+                    try {
+                        return mti.getObjectClassDefinition(pid, locale.toString())
+                                .getName();
+                    } catch (IllegalArgumentException e) {
+                        // MetaTypeProvider.getConnectionType might throw illegal
```

Comment:
Do we need to log at debug when this happens? Doesn't look like it currently and you still get the config showing up based on the comment